tests/test_connection_class.py

```diff
@@ -21,7 +21,7 @@
 # These are pretty arbitrary
 TEST_SERVER_AMS_NET_ID = "127.0.0.1.1.1"
 TEST_SERVER_IP_ADDRESS = "127.0.0.1"
-TEST_SERVER_AMS_PORT = pyads.PORT_SPS1
+TEST_SERVER_AMS_PORT = pyads.PORT_TC3PLC1
```

Comment:
I don't know if this works. I think the testserver runs on PORT_SPS1 by default so also in the tests. I don't see a reason to change this.